_posts/2020/2020-10-19-3-tipps-for-feedback-in-a-Scrum-Team

```diff
@@ -0,0 +1,81 @@
+As a Scrum Master I have to stress that you may speak to your Scrum Master / Agile Coach about the matter, but it’s about you to address your feedback. 
+The Scrum Master cannot give background about what exactly annoyed you. 
+Three team members telling someone they did not like a behaviour is ten times more efficient that one consolidated Scrum Master feedback. 
+The same is true for feedback via your manager. 
+If you complain to the manager directly instead of telling the person, this may result in a negative view your manager will get on that person, lesser likelihood for raises and usually, it will take longer for the manager to give feedback than for you. 
+What would you prefer? Would you like your colleagues to do give you direct feedback or go through your manager instead?
+
+## 3)	Praise often
+
+This article is mainly focussed on giving negative feedback, but of course the same is true for praise. 
+Honest and specific praise costs you very little effort and has a huge effect. 
+This is especially true when you had talked about an improvement your colleague could make and when he actually changed the behaviour, instant praise is the best way to reinforce this.
+
+
+## How we do it at ePages
+
+First and foremost, ePages has a company culture, that values openness, honesty and direct feedback. The onboarding for new employees is characterized by often and early feedback from different levels (HR, peers and manager) and of course the new employee also gives feedback to us and to the onboarding process. All employees participate in a 360 degree feedback process at least once a year, in this process everyone is rated on Communication, Collaboration and Quality of work and peers and manger give comments on what possible improvements they see.
+To be “rated” by your peers sounds worse that it is, in practice the comments your colleagues and managers give are generally appreciative and very helpful.
+But of course, giving feedback once a year is not enough in a self-organized team. 
+Therefor the Scrum Masters encourage giving feedback often. 
+To support the teams in actually doing so, we first have a look at what the team needs. 
+This may go from offering a training on the principles of feedback to doing regular feedback retrospectives. 
+Feedback is a key element of a retrospective, but of course not the only one. 
+Therefore, we also use this time slot for giving one on one feedback in pairs. 
+The teams liked this so much that we now do this regularly every 2 to 3 months. 
+It is a nice way to have some appointed time to make sure that you actually tell your colleagues what you liked and did not like.
+
+How do you ensure that feedback in your team is given early and often? 
+When was the last time you gave feedback to your colleagues be it positive or negative directly after the event? 
```

Comment:
"When was the last time you gave feedback to your colleagues - be it positive or negative - directly after the event?"